regalia-gson/build.gradle

```diff
@@ -1,16 +1,18 @@
 apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-kapt'
+apply plugin: 'kotlin-android-extensions'
```

Comment:
This isn't needed

Reply:
Sorry for the confusion, `kotlin-android-extensions` and `kotlin-kapt` not needed